packages/sdk/src/step-executor.ts

```
@@ -452,14 +452,18 @@ Return JSON only.`,
-    }
+    const rawContent = response.message.content ?? "";
+    const parsedCandidate = this.tryParseStructuredContent(rawContent) ?? rawContent;
+    const parsed = this.parseStepOutputContract(
+      {
+        ...augmentedStep,
+        output: {
+          path: judgeConfig.output_path,
+          ...(judgeConfig.output_schema ? { schema: judgeConfig.output_schema } : {}),
+        },
+      },
+      parsedCandidate,
```

ESLint

[error] Delete `,`

(prettier/prettier)